### ISSUE TITLE
Use Ollama running remote server

### ISSUE THREAD

**Issue by myrulezzz** [PromtEngineer/localGPT-Vision#20]
How can i use ollama running on remote server? Is there any way i can setup base url for ollama?

**Comment by saeed-siddiqui** [comment]
Yes you can run remotely and you can run http tunnel localhost to the remote port.

**Comment by myrulezzz** [comment]
How can i set it to run remotely?

**Comment by smileyboy2019** [comment]
I don't quite understand, is the URL of Olllam configured in the configuration file or do we need to modify the source code

**Comment by ImayaBharathi** [comment]
The feature i want to contribute is to add ollama support for loading Models. Since a threat is open for ollama, thought of asking here itself. @PromtEngineer

**Comment by PromtEngineer** [comment]
@ImayaBharathi Yes that will be extremely helpful and would really appreciate the contribution here.

**Comment by ImayaBharathi** [comment]
@PromtEngineer awesome, could you assign this issue to me. i will finish and raise a PR

**Comment by PromtEngineer** [comment]
@ImayaBharathi done.

**Comment by ImayaBharathi** [comment]
@PromtEngineer I went through the project and here is my understanding
Currently, the project supports local Ollama instances (localhost:11434) but doesn't allow connecting to remote Ollama servers. This addition would be valuable for users who:
- Run Ollama on dedicated GPU machines
- Use shared team Ollama instances
- Deploy Ollama in cloud/server environments

Proposed Changes:
1. Add remote URL configuration via:
   - Environment variable: OLLAMA_HOST
   - Settings page in web interface
2. Enhance error handling for remote connections
3. Add connection status checking and feedback

The implementation will:
- Be minimally invasive to existing code
- Maintain backwards compatibility (defaults to localhost)
- Follow project's error handling patterns
- Add clear documentation

Sample usage would be:
```bash
# Via environment variable
export OLLAMA_HOST='http://your-server:11434'
python app.py

# Or configure through web interface in Settings page

does this look good to you, if yes respond with `yes` then i would go ahead with the implementation. if not could you provide your inputs (If you’re short on time, even minimal feedback would be helpful.)
